mobile/native/src/event/api.rs

```diff
@@ -35,10 +37,23 @@ impl From<EventInternal> for Event {
             EventInternal::PositionUpdateNotification(position) => {
                 Event::PositionUpdateNotification(position.into())
             }
+            EventInternal::PositionCloseNotification(contract_symbol) => {
+                Event::PositionClosedNotification(PositionClosed { contract_symbol })
+            }
         }
     }
 }
 
+/// Wrapper struct needed by frb
+///
+/// The mirrored `ContractSymbol` does not get picked up correctly when using it directly as
```

Comment:
❓hmm, what error do you get when you try to mirror the `ContractSymbol` enum?

Reply:
I reported this separate when I initially opened the PR and pinged you there as well :)
Eventually I found a way to overcome it which is wrapping the struct. I don't have the exact error anymore and would not invest time into solving this at this point :)

If you are keen to see the error you can:

- Remove the wrapper struct, use `trade::ContractSymbol` directly in the enum varient directly.
- run `just gen`
- run clippy, the problem will be reported in the generated code